HW3/Menues/Database/Books.cs

```diff
@@ -112,18 +112,16 @@ public void Update(SqlConnection connection)
         public void Read(SqlConnection connection)
         {
             connection.Open();
-            string query = @"Select * from Books";
+            string query = @"select * from Books"; // You should write all commands low-case or UPPER-CASE
 
             using (SqlCommand comm = new SqlCommand(query, connection))
+            using (SqlDataReader read = comm.ExecuteReader()) // There was too many deep of brackets. Now it's prettier 
```

Comment:
mmm.. prettier - yes, but for me it's easier to read with brackets

Reply:
Left Igor's style